src/index.js

```diff
@@ -48,4 +48,6 @@ var namespaces         = 0
 	return getter;
 }
 
-module.exports = { getNamespace, getCountNamespaces, startRequest, bind, patch };
+module.exports = global.requestLocalStorage || { getNamespace, getCountNamespaces, startRequest, bind, patch };
+
+if (!global.requestLocalStorage) global.requestLocalStorage = module.exports;
```

Comment:
This seems a little risky.  If I have two packages that depend on different versions of `RequestLocalStorage` then one or the other of them is in for a surprise.  This package has been pretty stable for a while now, but if we ever find and fix a bug or add a useful feature then this patch could lead to maddening behavior.

Really `RequestLocalStorage`, like `React`, needs to be a singleton.  Currently it throws an explicit _error_ if multiple copies are used.  This patch would make it succeed with undefined behavior.

I appreciate the convenience this provides, though.  What about a version check?  If the global copy is the same version as the currently initializing copy then return a reference to the global.  Otherwise throw a _require time_ error?